### DISCUSSION TITLE
Adding a package to a workspace (equivalent to "yarn add")?

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [npm/feedback#224] [CLI] [answered]
Does NPM 7 offer a CLI command to add a package to a workspace?

There's a discussion here at Stack Overflow indicating nobody really knows the answer:

https://stackoverflow.com/questions/65237564/npm-7-workspaces-how-to-install-new-package-in-workspace

I've tried a bunch of search terms in this repo and not found anything either.

If there is no equivalent command to `yarn add` / `lerna add`, is there a "canonical" method for doing this right now with NPM 7? Pass a bunch of flags, edit `package.json` by hand, [install and use `lerna@4`](https://github.com/lerna/lerna/issues/2703#issuecomment-777089520) despite [valid questions about the current state of Lerna](https://github.com/lerna/lerna/issues/2703)?

I'd just like to understand how we're intended to be using NPM 7 with workspaces today, given its general availability.

Thanks!

**Comment by MylesBorins** [comment, marked as answer]
hey @aaronadamsCA

We have an open RFC that includes implementation of `npm workspace add`

https://github.com/npm/rfcs/pull/273
